src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -159,6 +159,7 @@ internal enum BoundKind: byte
         Call,
         EventAssignmentOperator,
         Attribute,
+        UnboundObjectCreationExpression,
```

Comment:
>UnboundObjectCreationExpression [](start = 8, length = 31)

please look around for uses of `BoundKind.ObjectCreationExpression`. I see that is it used in `BindComImportCoClassCreationExpression` for instance. Does `ImplicitObjectCreationExpression` need to be handled there (or would it have already been replaced?)

Same question for `HasExpressionSymbols` (maybe we can add an `Assert` there?)
Same question for `NullableAlwaysHasValue` and `NullableNeverHasValue`